src/Introspection/__Type.hack

```diff
@@ -10,31 +10,31 @@ use Slack\GraphQL;
 interface __Type {
 
     <<GraphQL\Field('kind', 'Kind of the type')>>
-    public function getKind(): __TypeKind;
+    public function getIntrospectionKind(): __TypeKind;
```

Comment:
nice, i like this for clarity

Reply:
i think this is also needed more functionality too - for `TNonNullableType` these mostly return null, whereas for `TNullableType` they delegate to the inner type's `getName()`, `getDescription()`, `getOfType()`, etc, methods.

Reply:
YEa, I hadn't read that far 😄